From: Yaroslav Halchenko <debian@onerussian.com>
Subject: comment out msgpack and duecredit in setup.py

  duecredit is absent on some elderly releases, and anyways
  not clear yet even why it was needed 

  msgpack -- for some reason on older systems even if installed
  system-wide from debian package, still attempts to pip install it
  for no good reason afaik

Last-Update: 2018-03-03

--- a/setup.py
+++ b/setup.py
@@ -74,7 +74,7 @@ requires = {
     (['colorama'] if on_windows else []),
     'downloaders': [
         'boto',
-        'msgpack',
+        #'msgpack',
         'requests>=1.2',
     ] + keyring_requires,
     'downloaders-extra': [
